windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -19,6 +19,18 @@ You must have a subscription to receive support for the Red Hat WMCO. Deploying
 
 include::modules/wmco-prerequisites.adoc[leveloffset=+1]
 
+[id="wmco-2-0-3"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.3
+
+Issued:2021-07-28
+
+The WMCO 2.0.3 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2021:2926[RHBA-2021:2926].
+
+[id="wmco-2-0-3-bug-fixes"]
+=== Bug fixes
+
+* This WMCO release fixes a bug that prevented users from upgrading to WMCO 3.0.0. Users should upgrade to WMCO 2.0.3 before upgrading to {product-title} 4.8. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1985349[**BZ#1985349**])
```

Comment:
Just wondering if the last sentence should add that OCP 4.8 uses WMCO 3, so they can see the correlation? like "... before upgrading to {product-title} 4.8, which uses|supports|\<some-other-word> WMC 3" ?